pulsar-common/src/main/java/org/apache/pulsar/common/util/collections/BitSetRecyclable.java

```diff
@@ -105,6 +105,35 @@ private void recalculateWordsInUse() {
         wordsInUse = i+1; // The new logical size
     }
 
+    public int compareTo(BitSetRecyclable bitSetRecyclable) {
+        long[] words1 = toLongArray();
```

Comment:
I suggest we avoid this memory copy in `toLongArray`, just use `words` and `wordsInUse` instead.

Reply:
Fixed. @Jason918 
PTAL,thanks!

Reply:
remove compareTo method，use nextSetBit instead of compareTo！ @Jason918 PTAL,thanks!